plugins/cicd-statistics/api-report.md

```diff
@@ -0,0 +1,182 @@
+//
+// @public
+export interface CicdStatisticsApi {
+  // (undocumented)
+  fetchBuilds(options: FetchBuildsOptions): Promise<CicdState>;
+  // (undocumented)
+  getConfiguration(): Promise<Partial<CicdConfiguration>>;
+}
+
+// Warning: (ae-missing-release-tag) "cicdStatisticsApiRef" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const cicdStatisticsApiRef: ApiRef<CicdStatisticsApi>;
+
+// Warning: (ae-missing-release-tag) "cicdStatisticsPlugin" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const cicdStatisticsPlugin: BackstagePlugin<
+  {
+    entityContent: RouteRef<undefined>;
+  },
+  {}
+>;
+
+// Warning: (ae-forgotten-export) The symbol "EntityPageCicdCharts" needs to be exported by the entry point index.d.ts
+// Warning: (ae-missing-release-tag) "EntityCicdStatisticsContent" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const EntityCicdStatisticsContent: EntityPageCicdCharts;
+
+// Warning: (ae-missing-release-tag) "FetchBuildsOptions" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface FetchBuildsOptions {
+  // (undocumented)
+  abortSignal: AbortSignal;
+  // (undocumented)
+  entity: Entity;
+  // (undocumented)
+  filterStatus: Array<FilterStatusType<'all'>>;
+  // (undocumented)
+  filterType: FilterBranchType<'all'>;
+  // (undocumented)
+  timeFrom: Date;
+  // (undocumented)
+  timeTo: Date;
+  // (undocumented)
+  updateProgress: UpdateProgress;
+}
+
+// Warning: (ae-missing-release-tag) "FilterBranchType" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export type FilterBranchType<Extra extends string = never> =
+  | Extra
+  | 'master'
+  | 'branch';
+
+// Warning: (ae-missing-release-tag) "FilterStatusType" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export type FilterStatusType<Extra extends string = never> =
+  | Extra
+  | 'unknown'
+  | 'enqueued'
+  | 'scheduled'
+  | 'running'
+  | 'aborted'
+  | 'succeeded'
+  | 'failed'
+  | 'stalled'
+  | 'expired';
+
+// Warning: (ae-missing-release-tag) "rootCatalogCicdStatsRouteRef" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const rootCatalogCicdStatsRouteRef: RouteRef<undefined>;
+
+// Warning: (ae-missing-release-tag) "Stage" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface Stage {
```

Comment:
Think it would be worth calling this `BuildStage` for a bit of extra context? could see this being just fine too tho

Reply:
Yeah I kind of thought so too. It might be more clear. A Stage can have sub stages though, which are also of type Stage. BuildStage *does* (to me) sounds like the top-level stages.

I'm fine with either, happy to change if it makes more sense.